features/common.branding.v1/providers/theme-provider.tsx

```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+
+import { ThemeProvider as OxygenThemeProvider } from "@oxygen-ui/react/theme";
+import React, { PropsWithChildren, ReactElement, useMemo } from "react";
+import { Helmet } from "react-helmet";
+import { generateTheme } from "../branding/theme";
+import { ThemeProviderContext, ThemeProviderContextProps } from "../contexts/theme-provider-context";
+import { ThemePreferenceMeta } from "../meta";
+import { BrandingPreferenceAPIResponseInterface } from "../models";
+
+/**
+ * Props interface for the ThemeProvider.
+ */export interface ThemeProviderProps {
```

Comment:
Check formatting here.
```suggestion
/**
 * Props interface for the ThemeProvider.
 */
export interface ThemeProviderProps {
```

Reply:
addressed with https://github.com/wso2/identity-apps/pull/6173/commits/651b7a87b8b4c871d6af2f41362a7f8759aa1121